src/CleanSolutionTemplate.Domain/Common/Entity.cs

```diff
@@ -4,29 +4,17 @@ namespace CleanSolutionTemplate.Domain.Common;
 
 public abstract class Entity
 {
-    private readonly List<DomainEvent> _domainEvents = new();
+    readonly List<DomainEvent> _domainEvents = new();
```

Comment:
```suggestion
    private readonly List<DomainEvent> _domainEvents = new();
```

It's a matter of style